numpy/_core/src/umath/loops_umath_fp.dispatch.c.src

```diff
@@ -258,4 +278,37 @@ NPY_NO_EXPORT void NPY_CPU_DISPATCH_CURFX(@TYPE@_@func@)
     }
 }
 /**end repeat1**/
+
+/**begin repeat1
+ *  #func = arctan2#
+ *  #intrin = atan2#
```

Comment:
this looks unrelated to pow

Reply:
the earlier loop was templated to do both `pow` and `atan2` and it was getting hard to read. I suggested splitting them into their own sections, ,